src/databricks/labs/ucx/framework/crawlers.py

```diff
@@ -41,6 +41,27 @@ def _schema_for(cls, klass):
             fields.append(f"{f.name} {spark_type}{not_null}")
         return ", ".join(fields)
 
+    @classmethod
+    def _filter_none_rows(cls, rows):
+        results = []
+        nullable_fields = []
+
+        for field in dataclasses.fields(rows[0]):
+            if field.default is None:
+                nullable_fields.append(field.name)
+
+        for row in rows:
+            row_contains_none = False
+            for column, value in dataclasses.asdict(row).items():
+                if value is None and column not in nullable_fields:
+                    logger.debug(f"Field {column} is None, filtering row")
```

Comment:
```suggestion
                    logger.warning(f"[{full_table_name}] Field {column} is None, filtering row")
```

also propagate the `full_table_name` down here, so that the logs are more clear.